### PR TITLE
fix: add zk_kafka group to hosts template

### LINKED ISSUE #46: zk_kafka group is missing from the hosts template

### PR DESCRIPTION
**Pull request by leopaul36**
Fix #46

### CODE REVIEW

File: roles/ansible-tdp-common-actions/templates/ansible-hosts.j2 [hunk 1]
```diff
@@ -11,6 +11,9 @@ edge-01
 [zk]
 master-0[1:3]
 
+[zk_kafka]
```
Comment (rpignolet) [inline roles/ansible-tdp-common-actions/templates/ansible-hosts.j2:14]: What is this group ? In the README, the dedicated ZK for Kafka is on the `zk` host.

Comment (leopaul36) [reply roles/ansible-tdp-common-actions/templates/ansible-hosts.j2:14]: It won't be needed, this fix is wrong as told by @Nuttymoon [here](https://github.com/TOSIT-IO/tdp-getting-started/pull/136#issuecomment-1190096329)

I will open a new PR with the correct fix

**Comment by leopaul36** [comment]
Not needed as stated [here](https://github.com/TOSIT-IO/tdp-getting-started/pull/136#issuecomment-1190096329)

File: roles/ansible-tdp-common-actions/templates/ansible-hosts.j2 (hunk 1 above)
Comment (rpignolet) [reply roles/ansible-tdp-common-actions/templates/ansible-hosts.j2:14]: Yes I read my mails after I comment this PR.

**Pull request closed without merging**